guides/common/attributes-satellite.adoc

```diff
@@ -136,27 +136,28 @@
 
 // For a release
 :RepoSatelliteVersion: {ProjectVersion}
+:project-client-2-name: satellite-6-client-2
 
 // RHEL 9 Satellite repos
 :RepoRHEL9ServerSatelliteCapsuleProjectVersion: satellite-capsule-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
 :RepoRHEL9ServerSatelliteMaintenanceProjectVersion: satellite-maintenance-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
 :RepoRHEL9ServerSatelliteServerProjectVersion: satellite-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
-:RepoRHEL9ServerSatelliteToolsProjectVersion: satellite-client-6-for-rhel-9-<arch>-rpms
+:RepoRHEL9ServerSatelliteToolsProjectVersion: {project-client-2-name}
```

Comment:
I'm pretty sure there's one repo built for each RHEL version so I suspect it'll something like this:
```suggestion
:RepoRHEL9ServerSatelliteToolsProjectVersion: {project-client-2-name}-for-rhel-9-<arch>-rpms
```

@ehelms can you confirm?

Reply:
Confirmed